releasenotes/notes/quantum_shannon_decomp-facaa362a3ca8ba3.yaml

```diff
@@ -1,13 +1,12 @@
 ---
 features:
   - |
-    Added a new module to apply Quantum Shannon Decomposition of
-    arbitrary unitaries. This functionality replaces the previous
-    usage of the isometry module in the unitary synthesis transpiler
-    pass as well as when adding unitaries to a circuit using a
-    UnitaryGate.
-upgrade:
```

Comment:
I think I might have asked for an upgrade note in the review that the default synthesis output would change after the release and was planning to expand this with instructions on how to manually use the isometry module to recreate the previous behavior. I'll add a note for this when we do the final round up before the final 0.21.0 release.